langchain/vectorstores/mongodb_atlas_vector_search.py

```diff
@@ -0,0 +1,247 @@
+from __future__ import annotations
+
+import logging
+from typing import Any, Callable, Iterable, List, Optional, Tuple, Dict
+
+from langchain.docstore.document import Document
+from langchain.embeddings.base import Embeddings
+from langchain.vectorstores.base import VectorStore
+
+try:
```

Comment:
this should not be imported here, should only do `if TYPE_CHECKING`

Reply:
fixed